### ISSUE TITLE
hydrator dupe 1.12.2

### ISSUE THREAD

**Issue by MasterSloth1** [PrinceOfAmber/Cyclic#673]
Minecraft version & Mod Version:
forge - 14.23.1.2607
minecraft - 1.12.2
cyclic - 1.12.2-1.11.1

Single player or Server:
server

Describe problem (what you were doing / what happened):
placed item into the bottom left slot (red concrete powder) and it continued to make concrete but the powder did not get used.

Crash log (if any):
null

**Comment by PrinceOfAmber** [comment]
Yeah good find! It was trying to drain the recipe cost from top left only!

**Comment by PrinceOfAmber** [comment]
Resolved in https://minecraft.curseforge.com/projects/cyclic/files/2530166

**Comment by artdude543** [comment]
Seems this is still a valid issue in 1.14.9 @PrinceOfAmber 

https://i.artdude.uk/jo5Jhdir.mp4

**Comment by PrinceOfAmber** [comment]
Sorry i didnt notice the update to to this for some reason, must have got lost in the github emails . on my radar now

**Comment by PrinceOfAmber** [comment]
@artdude543  how did you even get a recipe of size 2 in the hydrator?

I dont know why i coded it this way, but when i use script

`Hydrator.addRecipe(<minecraft:gravel>, [<minecraft:sand>,<minecraft:sand>], 50);`



i get error in log

` Error executing {[0:crafttweaker]: hyd.zs}: Input array must be length 1 or length 4`


Assuming that grey block in video is hardened_clay, it looks like its it is using that shapeless recipe of 1 grey sand to 1 clay.  BUT theres still a bug here, its assuming the top left slot has ingredient, so it should be reducing one from your lower left stack in that video

**Comment by PrinceOfAmber** [comment]
If its not too much work ill see about allowing size 2/3 recipes, no idea why i had it that way
